src/coreclr/debug/daccess/task.cpp

```diff
@@ -5225,7 +5225,7 @@ EnumMethodInstances::Next(ClrDataAccess* dac,
         }
     }
 
-    if (!(g_pEEInterface->GetFunctionAddress(m_methodIter.Current()) != NULL))
+    if (!m_methodIter.Current()->HasNativeCodeReJITAware())
```

Comment:
These were places where I think we want to search for any version of the code, not just the two versions that this function looks at. If you don't have time to fix it then we should at least comment that is a workaround and not the ideal behavior.

Reply:
HasNativeCodeReJITAware uses the function GetNativeCodeReJITAware, which I believe now searches for any version of the code. It gets all the ILVersions of the code and then for each ILVersion, gets each NativeVersion, and will return the first NativeCodeVersion it finds. (see line 937 of method.cpp). If this code is not searching for any version of the code, please explain to me what needs to be done differently, and I am happy to do it.

Reply:
I discussed with @mikelle-rogers in person, but recording here for anyone else reading in the future. I was mistaken in my comment above and didn't realize Mikelle had already changed the behavior of HasNativeCodeReJITAware() to do an enumeration of all versions.